### PR TITLE
8 new playlist page

### LINKED ISSUE #8: New Playlist Page
**WHY**

The user must be able to create a new playlist.

**WHERE**

On the NewPlaylistPage

**HOW**

- [x] Add a form with one input (name of the playlist) and one button submit
- [x] Add a state input (str)
- [x] Add an handleChange function triggering on onChange input
- [x] Add an handleSubmit function triggering on onSubmit form : api post request playlist#create, if request validated redirect to PlaylistShowPage
- [x] Add style and design
- [x] test locally
- [ ] run eslint
- [ ] update readme
- [x] rebase on master

**DOC**

none

### PR DESCRIPTION
**Pull request by raphaelchpprt**
**Related issues**

closes #8 

**Changes**

:heavy_plus_sign: form for new playlist with material ui component

:heavy_plus_sign: css

:heavy_plus_sign: PoppinsBoldItalic in _variables.scss

:heavy_plus_sign: api request indications as comments in onSubmit form and route

* * *
- [x] I ran this code locally

### CODE REVIEW

File: src/pages/NewPlaylist/index.jsx [hunk 1]
```diff
@@ -1,11 +1,56 @@
-import React from "react";
+import React, { useState } from "react";
+import "./styles.scss";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+
+const useStyles = makeStyles((theme) => ({
```
Comment (julienemo) [inline src/pages/NewPlaylist/index.jsx:7]: I don't know this very well, but I wonder whether there is a way to put this into the css :thinking:

File: src/pages/NewPlaylist/index.jsx [hunk 2]
```diff
@@ -1,11 +1,56 @@
...
+  },
+}));
+
+const NewPlaylist = () => {
+  const classes = useStyles();
+  const [input, setInput] = useState();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(input);
```
Comment (julienemo) [inline src/pages/NewPlaylist/index.jsx:23]: I suggest we control whether input is empty/without wording chars (ex: if someone names his playlist (-'&""'(-_-('"), do we let him pass ?)

Comment (julienemo) [reply src/pages/NewPlaylist/index.jsx:23]: Also could redirect to /temp-playlist just for the sake of test ?

Comment (raphaelchpprt) [reply src/pages/NewPlaylist/index.jsx:23]: I suggest special characters allowed but the request is only triggered if the input is not an empty string. For the test link, I can add that

File: src/pages/NewPlaylist/index.jsx (hunk 1 above)
Comment (raphaelchpprt) [reply src/pages/NewPlaylist/index.jsx:7]: If we want to set a color theme, it's better to leave it I guess

**Pull request merged**